src/selection.ts

```diff
@@ -529,7 +534,7 @@ export class DataSelection {
         // triggers creation of the deeptable column as a side-effect.
         return tile.get_column(this.name);
       }),
-    ).then(() => {});
+    ).then(() => { });
```

Comment:
Ensure that you are running `npx pretty-quick` or equivalent and no other linters on javascript